### PR TITLE
Group acl integration

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
PR for integration testing the Group ACL functionality in `permission-set`

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
(Rebased on latest `develop`, you may need to do a `git reset --hard origin/group-acl-integration` if a `git pull` results in conflict.)

File: package.json [hunk 1]
```diff
@@ -64,7 +65,7 @@
     "recursive-readdir": "^2.1.0",
     "rimraf": "^2.5.0",
     "solid-namespace": "^0.1.0",
-    "solid-permissions": "^0.5.2",
+    "solid-permissions": "solid/solid-permissions#feature-group-acl",
```
Comment (RubenVerborgh) [inline package.json]: Should be updated to release version.

(Note: in the future, you can just put the next release version here, and use `npm ln` locally.)

File: test/integration/acl-oidc-test.js [hunk 2]
```diff
@@ -499,6 +499,87 @@ describe('ACL HTTP', function () {
...
+    //     '<#Public> a <http://www.w3.org/ns/auth/acl#Authorization>;\n' +
+    //     ' <http://www.w3.org/ns/auth/acl#accessTo> <./>;\n' +
+    //     ' <http://www.w3.org/ns/auth/acl#agentGroup> <group-listing#folks>;\n' +
+    //     ' <http://www.w3.org/ns/auth/acl#mode> <http://www.w3.org/ns/auth/acl#Read> .\n'
+    //   request.put(options, function (error, response, body) {
+    //     assert.equal(error, null)
+    //     assert.equal(response.statusCode, 201)
+    //     done()
+    //   })
+    // })
```
Comment (RubenVerborgh) [inline test/integration/acl-oidc-test.js]: Let's clean that up.

File: test/resources/accounts-acl/tim.localhost/group/test-folder/group-listing.ttl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@prefix acl: <http://www.w3.org/ns/auth/acl#>.
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+
+<> a acl:GroupListing.
+
+<#folks>
+    a vcard:Group;
+    vcard:hasUID <urn:uuid:8831CBAD-1111-2222-8563-F0F4787EG398:ABGroup> ;
```
Comment (RubenVerborgh) [inline test/resources/accounts-acl/tim.localhost/group/test-folder/group-listing.ttl]: Not needed.

File: lib/handlers/allow.js [hunk 4]
```diff
@@ -39,6 +39,12 @@ function allow (mode) {
         origin: req.get('origin'),
         host: req.protocol + '://' + req.get('host'),
         fetch: fetchFromLdp(mapper, ldp),
+        fetchGraph: (uri, options) => {
+          // first try loading from local fs
+          return ldp.getGraph(uri, options.contentType)
+            // failing that, fetch remote graph
+            .catch(() => ldp.fetchGraph(uri, options))
```
Comment (RubenVerborgh) [inline lib/handlers/allow.js]: We'll probably need better logic for this in the future.

**Pull request merged**